### PR TITLE
Fut 12 app for meal energy

### PR DESCRIPTION
**Pull request by karollayna**

### CODE REVIEW

File: Future/inst/shiny/meal app/app_for_meal/app.R [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+               div(id = "box01", box(
+                   width = 12,
+                   column(6,
+                          pickerInput(
+                              inputId = "product01",
+                              label = "Wybierz produkt",
+                              choices = "",
+                              options = list(`live-search` = TRUE)
+                          )),
+
```
Comment (darsoo) [inline Future/inst/shiny/meal app/app_for_meal/app.R:21]: remove line

File: Future/inst/shiny/meal app/app_for_meal/app.R [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+                   ),
+                   shinyWidgets::actionBttn(
+                       inputId = "remove03",
+                       label = "Usun produkt",
+                       size = "sm",
+                       style = "jelly",
+                       color = "danger"
+                   ),
+               ))),
+               hidden(div(
```
Comment (darsoo) [inline Future/inst/shiny/meal app/app_for_meal/app.R:104]: be consistent - use one style:  
`hidden(div(box` 
or
`hidden(div(
box(`

File: Future/inst/shiny/meal app/app_for_meal/app.R [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+                           color = "primary"
+                       ),
+                       shinyWidgets::actionBttn(
+                           inputId = "remove04",
+                           label = "Usun produkt",
+                           size = "sm",
+                           style = "jelly",
+                           color = "danger"
+                       ),
+
```
Comment (darsoo) [inline Future/inst/shiny/meal app/app_for_meal/app.R:139]: remove line

File: Future/inst/shiny/meal app/app_for_meal/app.R [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+                           )
+                       ),
+                       shinyWidgets::actionBttn(
+                              inputId = "remove05",
+                              label = "Usun produkt",
+                              size = "sm",
+                              style = "jelly",
+                              color = "danger"
+                          ),
+
```
Comment (darsoo) [inline Future/inst/shiny/meal app/app_for_meal/app.R:168]: remove line

File: Future/inst/shiny/meal app/app_for_meal/app.R [hunk 5]
```diff
@@ -0,0 +1,283 @@
...
+               ))),
+
```
Comment (darsoo) [inline Future/inst/shiny/meal app/app_for_meal/app.R:170]: remove line

File: Future/inst/shiny/meal app/app_for_meal/app.R [hunk 6]
```diff
@@ -0,0 +1,283 @@
+library(shiny)
+
+
+header <- dashboardHeader(title = "Future")
+
+sidebar <- dashboardSidebar()
```
Comment (darsoo) [inline Future/inst/shiny/meal app/app_for_meal/app.R:6]: until we have only one tab, maybe we can remove sidebar?

**Review by darsoo — APPROVED** [review]

File: Future/inst/shiny/meal app/app_for_meal/app.R (hunk 1 above)
Comment (karollayna) [reply Future/inst/shiny/meal app/app_for_meal/app.R:21]: done

File: Future/inst/shiny/meal app/app_for_meal/app.R (hunk 2 above)
Comment (karollayna) [reply Future/inst/shiny/meal app/app_for_meal/app.R:104]: done

File: Future/inst/shiny/meal app/app_for_meal/app.R (hunk 3 above)
Comment (karollayna) [reply Future/inst/shiny/meal app/app_for_meal/app.R:139]: done

File: Future/inst/shiny/meal app/app_for_meal/app.R (hunk 4 above)
Comment (karollayna) [reply Future/inst/shiny/meal app/app_for_meal/app.R:168]: done

File: Future/inst/shiny/meal app/app_for_meal/app.R (hunk 5 above)
Comment (karollayna) [reply Future/inst/shiny/meal app/app_for_meal/app.R:170]: done

File: Future/inst/shiny/meal app/app_for_meal/app.R (hunk 6 above)
Comment (karollayna) [reply Future/inst/shiny/meal app/app_for_meal/app.R:6]: we can't - tagAssert

Comment (darsoo) [reply Future/inst/shiny/meal app/app_for_meal/app.R:6]: https://stackoverflow.com/questions/35170065/hide-sidebar-in-default-in-shinydashboard

Comment (karollayna) [reply Future/inst/shiny/meal app/app_for_meal/app.R:6]: done

**Pull request merged**